agent/tools/tools_test.go

```diff
@@ -152,7 +152,7 @@ func (t *ToolsSuite) TestReadToolsErrors(c *gc.C) {
 	vers := version.MustParseBinary("1.2.3-precise-amd64")
 	testTools, err := agenttools.ReadTools(t.dataDir, vers)
 	c.Assert(testTools, gc.IsNil)
-	c.Assert(err, gc.ErrorMatches, "cannot read tools metadata in tools directory: .*")
+	c.Assert(err, gc.ErrorMatches, "cannot read agent binaries metadata in the expected directory: .*")
```

Comment:
here I would probably say "agent metadata". And given we're listing a path, does it make sense to just say: 
  in the directory: /path

If we do decide to change it, we'd need to update at least 3 other places where we check this string. But I'll skip putting 'ditto' on all of them. Unless you'd prefer it.

Reply:
I am not sure that we put a directory path... We put in an error which may or may not contain the directory. I'll change the error to contain path as a drive-by here :D